client/src/components/Search/index.js

```diff
@@ -94,3 +102,16 @@ export default class Search extends Component {
     );
   }
 }
+
+function average(...nums) {
```

Comment:
What is the point of this function? This looks like code snippet from the Udacity course...